docs/Plasma-Mobile-on-Unleashed.md

```diff
@@ -59,7 +63,6 @@ Follow the top level README to flash the image to an SD card (you will probably
 
 ## Known issues
 
- * Qt5 base will most likely fail to cross compile for RISC-V. As a work around you can apply the patch from here: https://github.com/meta-qt5/meta-qt5/pull/156/commits
- * Sometimes the PCIe devices do not work on first boot. In you can't detect the PCIe devices on the board try power cycling the board and then they should appear.
+  * Sometimes the PCIe devices do not work on first boot. In you can't detect the PCIe devices on the board try power cycling the board and then they should appear.
```

Comment:
s/In you can't/In case you can't/

Reply:
thanks for pointing it out, can you propose a fix ? this has already been merged.

Reply:
Done. Please see https://github.com/riscv/meta-riscv/pull/130